recipe/install-core.sh

```diff
@@ -22,24 +21,19 @@ popd
 echo "done"
 set -e
 
-cp -R * "$INSTALL_DIR"
-rm -rf "$INSTALL_DIR/obj"
-rm -rf "$INSTALL_DIR/pkg"
-rm "$INSTALL_DIR"/conda_build.sh
-cp "$RECIPE_DIR/gap.sh" "$PREFIX/bin/gap"
-chmod +x "$PREFIX/bin/gap"
+mkdir -p "$INSTALL_DIR"/pkg
+cp -R pkg/* "$INSTALL_DIR"/pkg
 
-GAP_SRC_PATH=`ls -d "$INSTALL_DIR"/bin/*/src`
-rm "$GAP_SRC_PATH"
-ln -s "$INSTALL_DIR"/src/ "$GAP_SRC_PATH"
+mkdir -p "$INSTALL_DIR"/src
+cp -R src/* "$INSTALL_DIR"/src
```

Comment:
It is not necessary anymore to install the source code
```suggestion
```

Reply:
Thanks